types/trigger.d.ts

```diff
@@ -136,7 +136,8 @@ export type TimelineTrigger<Data extends RaidbossData> = BaseTrigger<Data, 'None
 };
 
 // Because timeline functions run during loading, they only support the base RaidbossData.
-export type TimelineFunc = string | string[] | ((data: RaidbossData) => TimelineFunc);
+export type TimelineFunc = (data: RaidbossData) => TimelineField | undefined;
+export type TimelineField = string | (string | TimelineFunc)[];
```

Comment:
```suggestion
export type TimelineFunc = (data: RaidbossData) => TimelineField;
export type TimelineField = string | (string | TimelineFunc)[] | TimelineFunc | undefined;
```

I am not sure that there's an example, but there's no reason that `timeline` couldn't be a single throwaway func either.  Mostly, I am suggesting it so that this parallels `TriggerFunc`/`TriggerField`.  What do you think?

Reply:
Sounds good to me.